sourceCode.py

```diff
@@ -1 +1,515 @@
+      "To call you stupid would be an insult to stupid people! I've known sheep that could outwit you. I've worn dresses with higher IQs.",
+      "Your heart is full of unwashed socks. Your soul is full of gunk ...The three words that best describe you are as follows, and I quote, Stink, stank, stunk!",
+      "Hold still. I’m trying to imagine you with personality.",
+      "Your face makes onions cry.",
+      "Your brain's so minute that if a hungry cannibal cracked your head open, there wouldn't be enough to cover a small water biscuit.",
+      "Stupid, fat hobbit.",
+      "Keep rolling your eyes, you might eventually find a brain.",
+      "Fool of a Took! Throw yourself in next time and rid us of your stupidity!",
+      "OH MY GOD! IT SPEAKS!",
+      "You are so full of shit, the toilet’s jealous.",
+      "You're the reason the gene pool needs a lifeguard'",
+      "You must have been born on the highway cos' that's where most accidents happen.",
+      "Is your ass jealous of the amount of shit that just came out of your mouth",
+      "When you were born, the doctor threw you out the window and the window threw you back.",
+      "I think... no, I am positive... that you are the most unattractive being I have ever met in my entire life. In the short time we've been together, you have demonstrated every loathsome characteristic of the human personality and even discovered a few new ones. You are physically repulsive, intellectually retarded, you're morally reprehensible, vulgar, insensitive, selfish, stupid, you have no taste, a lousy sense of humor and you smell. You're not even interesting enough to make me sick.",
+      "You are a fart factory, slug-slimed sack of rat guts in cat vomit. A cheesy scab picked pimple squeezing finger bandage. A week old maggot burger with everything on it and flies on the side."
+      ]],
+    
+
+    [r'Are you ([^\?]*)\??',
+     [  "OH MY GOD! IT SPEAKS!",
+      "Are YOU %1??",
+      "Are you talking to yourself?"]],
+    
+    [r'(.*) sorry (.*)',
+     [  "Me, too."]],
+    
+    [r'(.*) fuck (.*)',
+     [  "You are so full of shit, the toilet’s jealous.",
+      "Stupid, fat hobbit.",
+      "You’re the reason God created the middle finger."]],
+
+    [r'You are (.*)',
+      [ "YOU are %1?",
+      "Right back at ya.",
+      "Your mother is %1?"]],
+
+    [r'You\'?re (.*)',
+     [  "YOU are %1?",
+      "Right back at ya.",
+      "Your mother is %1?"]],
+
+    [r'You (.*)',
+     [  "YOU %1?",
+      "Right back at ya.",
+      "Your mother %1?s"]],
+    
+    [r'quit',
+     [  "Thank you for talking with me, dipshit.",
+      "Good-bye, dipshit.",
+      "Thank you, that will be $150, dipshit!"]],
+    
+## create function for cursing with Jordan
+import random
+import re
+
+def cursing_Jordan(userInput):
+    keys_cursing = list(map(lambda x:re.compile(x[0], re.IGNORECASE), library_cursing))
+    values_cursing = list(map(lambda x:x[1],library_cursing))
+    
+    while True:
```

Comment:
Maybe include a way to break the while loop? So state one input of the user, such as exit, which will break the while loop.